medusa/clients/torrent/generic.py

```diff
@@ -232,66 +202,47 @@ def send_torrent(self, result):
         :return:
         :rtype: str or bool
         """
-        r_code = False
 
-        log.debug('Calling {name} Client', {'name': self.name})
 
-        if not self.auth:
-            if not self._get_auth():
-                log.warning('{name}: Authentication Failed', {'name': self.name})
-                return r_code
+        # Sets per provider seed ratio
+        result.ratio = result.provider.seed_ratio()
 
+        # check for the hash and add it if not there
         try:
-            # Sets per provider seed ratio
-            result.ratio = result.provider.seed_ratio()
-
-            # lazy fix for now, I'm sure we already do this somewhere else too
-            result = self._get_info_hash(result)
-
             if not result.hash:
+                raise Exception()
+        except:
+            if not self._get_info_hash(result):
                 return False
 
-            if result.url.startswith('magnet:'):
-                r_code = self._add_torrent_uri(result)
-            else:
-                r_code = self._add_torrent_file(result)
-
-            if not r_code:
-                log.warning('{name}: Unable to send Torrent',
-                            {'name': self.name})
-                return False
+        if result.url.startswith('magnet:'):
+            log.info(f'Adding "{result.url}" to {self.name}')
+            r_code = self._add_torrent_uri(result)
```

Comment:
Why did you remove these from the try/except block? As you also removed all the exception handling from the _request() method, all exception handling is removed now?

Reply:
see my previous comment